lib/blacklight/solr/response/group_response.rb

```diff
@@ -51,9 +51,9 @@ def entry_name(options)
     )
   end
 
-  def method_missing meth, *args, &block
+  def method_missing(...)
```

Comment:
Is this compatible with all versions of ruby that we want to support?

Reply:
It sure is not 🤣.  I'll put it into .rubocop.yml instead.